### PR TITLE
metadata.tsv Edits

### PR DESCRIPTION
**Pull request by utku-ozturk**
Trello - https://trello.com/c/RKsnCBsy (completed)
Trello - https://trello.com/c/0NKCxX9E (completed)

### CODE REVIEW

File: src/encoded/batch_download.py [hunk 1]
```diff
@@ -89,6 +89,7 @@ def includeme(config):
     ('Generating Lab',              (FILE,      ['lab.display_title'], True)),
     ('Contributing Lab',            (FILE,      ['contributing_labs.display_title'], True)),
     ('Notes',                       (FILE,      ['notes_to_tsv'], True)),
+    ('URL',                         (FILE,      ['open_data_url'], True)),
```
Comment (aschroed) [inline src/encoded/batch_download.py]: Can we give this the name 'Open Data URL'?

**Review by aschroed — APPROVED** [review]
This looks pretty good.
One thing I noticed that may be out of the scope of this particular PR is that in the case where a file is produced by a 4DN pipeline - the track_and_facet.experimental_lab === contributing_labs or at least 1 of the values in contributing labs, which looks kind of funny.  I don't know how easy it would be to filter those cases out of the contributing_labs column.  

If it's not an easy fix it can become a separate task and not a huge deal.

**Comment by aschroed** [comment]
Oh and one other small change - Dataset should appear before condition in column ordering.  Thanks!

**Comment by utku-ozturk** [comment]
@aschroed, we have removed contributing lab's redundancy and also reordered dataset/condition columns. It is now on the hotseat.

**Comment by aschroed** [comment]
Utku - this looks great to me.  Thanks!  If you are happy with it I am.

**Comment by utku-ozturk** [comment]
> Utku - this looks great to me. Thanks! If you are happy with it I am.

Great! Thanks for your review! I'll merge it into master shortly.

**Pull request merged**